projects/hitl/software/README.md

```diff
@@ -0,0 +1,61 @@
+# Hardware in the Loop - Software
+
+## Intro
+
+This directory stores all code used to control and test with Olin Electric Motorsports' hardware-in-the-loop harness. It is organized into hitl, tests, scripts, docs, and artifacts (see below).
+
+Note: All paths in this README file are given relative to the `software` directory.
+
+For setup instructions and documentation, see the [project docs](awenstrup.github.io).
+
+**Project docs:** awenstrup.github.io
```

Comment:
We should chat at some point about moving your docs over to Outline! There is an API so we could use the autogenerated docs and create an integration to upload them via Outline's web API :)

Reply:
I'm intrigued, you able to carve out 15-20 minutes in tonight's meeting?

Reply:
Yep!